src/foraging_gui/Foraging.py

```diff
@@ -148,6 +153,32 @@ def __init__(self, parent=None,box_number=1,start_bonsai_ide=True):
             self._ReconnectBonsai()   
         logging.info('Start up complete')
     
+    def _load_rig_metadata(self):
+        '''Load the latest rig metadata'''
+        if not os.path.exists(self.rig_metadata_folder):
+            self._manage_warning_labels(self.MetadataWarning,warning_text='No rig metadata folder found!')
+            logging.warning('No rig metadata folder found!')
```

Comment:
I think if the rig_metadata folder does not exist we should make the folder

Reply:
Good catch!

Reply:
updated